uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/WorkDay.php

```diff
@@ -42,6 +45,56 @@ public static function create($params) {
   }
 
   /**
+   * Returns the value for the option value, of the Work Day Type option group,
+   * with the given name.
+   *
+   * @param string $optionName
+   *
+   * @return string
+   */
+  private static function getTypeValue($optionName) {
+    if(is_null(self::$workDayTypes)) {
+      self::$workDayTypes = array_flip(self::buildOptions('type', 'validate'));
+
+      // The option value values are stored as strings on the database, so we
+      // need to make sure the values will always be returned as strings, even
+      // if they're numeric.
+      array_walk(self::$workDayTypes, function(&$type) {
+        $type = (string)$type;
+      });
+    }
+
```

Comment:
Do we really need to cast to string since the values are stored as strings in the db and I presume they will be retrieved as such?